modules/objdetect/include/opencv2/objdetect/aruco_board.hpp

```diff
@@ -142,6 +142,19 @@ class CV_EXPORTS_W_SIMPLE CharucoBoard : public Board {
     CV_WRAP float getSquareLength() const;
     CV_WRAP float getMarkerLength() const;
 
+    /** @brief Given a board configuration and a set of detected markers, returns the corresponding
+     * image points and object points to call solvePnP()
+     *
+     * @param detectedCharuco List of detected charuco corners of the board.
+     * CharucoBoard support std::vector<Point2f> or Mat for charuco corners.
+     * @param detectedIds List of charuco identifiers for each corner.
+     * CharucoBoard support std::vector<int> or Mat.
+     * @param objPoints Vector of vectors of board marker points in the board coordinate space.
+     * @param imgPoints Vector of vectors of the projections of board marker corner points.
+     */
+    CV_WRAP void matchImagePoints(InputArray detectedCharuco, InputArray detectedIds,
+                                  OutputArray objPoints, OutputArray imgPoints) const;
+
```

Comment:
Why do you need dedicated method for it?

Reply:
removed